lib/screen/home/home_navigation_page.dart

```diff
@@ -188,14 +193,27 @@ class HomeNavigationPageState extends State<HomeNavigationPage>
         options: [
           OptionItem(
             title: 'featured_works'.tr(),
-            icon: SvgPicture.asset(
-              'assets/images/icon_set.svg',
-              colorFilter:
-                  const ColorFilter.mode(AppColor.white, BlendMode.srcIn),
-            ),
-            onTap: () {
-              Navigator.of(context)
-                  .popAndPushNamed(AppRouter.featuredWorksPage);
+            icon: const Icon(AuIcon.discover),
+            onTap: () async {
+              final artworks =
+                  await injector<FeralFileService>().getFeaturedArtworks();
+              if (!mounted) {
+                return;
+              }
+              final tokenIds = artworks.map((e) => e.indexerTokenId).toList()
```

Comment:
Should we handle case tokenIDs is empty?

Reply:
handled